staff/lisandro-bertoli/precourse/calculator/Index-pro.js

```diff
@@ -0,0 +1,135 @@
+
+var RESULTADOS = [];
+var CANTIDAD_NUMEROS;
+var NUMEROS;
```

Comment:
It will be a good idea change the name of the variables following the camelcase rules

Reply:
You are defintetly right!! I will get on this at some point for sure!